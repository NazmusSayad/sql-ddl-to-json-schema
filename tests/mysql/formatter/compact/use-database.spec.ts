import { getCompactFormat } from '../../parse-handler';
import { run } from '../../runner';

const sql = ['USE dbname;', 'USE `dbname`;'];

run(getCompactFormat, {
  'Compact formatter: Should use database.': {
    queries: [sql.join('')],
  },
});
