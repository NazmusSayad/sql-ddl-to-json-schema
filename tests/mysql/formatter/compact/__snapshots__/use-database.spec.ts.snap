// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Compact formatter: Should use database. 1`] = `[]`;
